content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,171 @@
+This tutorial is for someone that would like their transactions to be signed from multiple devices. In this workflow, we will be using a desktop, laptop, and a mobile phone to create a 2-of-3 multisig wallet. You can also use this workflow as a template to easily create another version of a multisig wallet.
+{{< /notice >}}
+
+{{< notice info >}}
+You must be signed in to your account in order to access the messaging system. <br/>
+You will need to import or create a key for each of the devices you wish to use in the wallet. These wallets can exist as foreign or software keys. You only need one device to create the setup. The keys cannot be the same. If you need help creating keys click <u>[here](/getting-started/createsoftwarekey/)</u>
+{{</ notice >}}
+
+{{< tabs >}}
+
+  {{< tab "Desktop" >}}
+  
+1. Open the chat feature located on the left side bar. <br/><br/>
+    
+<img src = desktopChatButton.png
+    alt = "Chat Button (Screenshot)"/>
+
+2. Click the "Start new message" button and type in your user email. You should see your name and email pop up below, click it!<br/><br/>
+
+<img src = desktopStartNewMessage.png
+    alt = "start new message button (Screenshot)"/>
+<img src = desktopNameSelected.png
+    alt = "Name selected (Screenshot)"/>
+
+3. Send a friendly message to yourself to initiate the chat room. After sending the message, you should see a new chat room with yourself!<br/><br/>
+
+<img src = desktopJoinedChatRoom.png
+    alt = "Wallet + button (Screenshot)"/>
+
+4. Click the "Create a collaborative wallet" button to initiate the multisig configuration setup.<br/>
+
+<img src = desktopCreateCollabWalletBttn.png
+    alt = "collaborative wallet question (Screenshot)"/>
+
+5. Name your wallet and click "Continue".<br/><br/>
+    
+<img src = desktopWalletName.png
+    alt = "Create collaborative wallet view (Screenshot)"/>
+
+6. Add the number of keys you will be using in your wallet. Then, add the required keys to sign a transaction.<br/><br/>
+
+<img src = desktopWalletConfig.png
+    alt = "Configure collaborative wallet (Screenshot)">
+
+7. Be sure to check your configuration before clicking "Continue".<br/><br/>
+
+<img src = desktopCurrentConfig.png
+    alt = "your current configuration (Screenshot)">
+
+8. You can now review other parameters of your wallet configuration, then continue.<br/><br/>
+
+<img src = desktopReviewWallet.png
+    alt = "Review collaborative wallet (Screenshot)">
+
+9. Assign the keys that are required for signing wallet transactions, and then continue.
+
+{{< notice tip >}}
+The key in green (Software) is an eligible key to sign with for your current device. Foreign keys are not eligible for signing locally and will need to be signed on the device they were imported or created on.
+{{< /notice >}}
+
+<img src = desktopAssignKeys.png
+    alt = "Assign key (Screenshot)">
+
+10. Ensure your local key is selected (for me, that is my Desktop Key) and click "Continue".<br/><br/>
```

Comment:
Step 9 and 10 are mutually exclusive right? Maybe we can make them "9a" & "9b" ?

Also I slightly prefer that we use 10 as the default method, since 9 depends on the syncing to have succeeded already - which might not be as reliable.